pyuvdata/uvdata.py

```diff
@@ -32,11 +32,12 @@ class UVData(UVBase):
     Currently supported file types: uvfits, miriad, fhd.
     Provides phasing functions.
 
-    Attributes:
-        UVParameter objects: For full list see UVData Parameters
-            (http://pyuvdata.readthedocs.io/en/latest/uvdata_parameters.html).
-            Some are always required, some are required for certain phase_types
-            and others are always optional.
+    Attributes
+    ----------
+    UVParameter objects: For full list see UVData Parameters
+        (http://pyuvdata.readthedocs.io/en/latest/uvdata_parameters.html).
+        Some are always required, some are required for certain phase_types
+        and others are always optional.
```

Comment:
I don't think this is getting rendered correctly:
![image](https://user-images.githubusercontent.com/4752917/58767641-56f96a00-8543-11e9-93b7-5c051eb485ff.png)
Perhaps "For a full list..." should start on the next line?

Reply:
Thanks!